### PR TITLE
Translate helm command with helm show in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>
Translate helm command with helm show in Korean 
- https://helm.sh/docs/helm/helm_show/

### CODE REVIEW

**Comment by bluefriday** [comment]
@singlemocha @jmnote 

helm show 와 관련된 5개 명령어에 대한 문서를 번역하였습니다.
리뷰 부탁드립니다.

File: content/ko/docs/helm/helm_show_all.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_all.md:50]:
```suggestion
      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
```

File: content/ko/docs/helm/helm_show_chart.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_chart.md:49]:
```suggestion
      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
```

File: content/ko/docs/helm/helm_show_readme.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_readme.md:50]:
```suggestion
      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
```

File: content/ko/docs/helm/helm_show.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Helm Show"
+---
+
+## helm show
+
+차트의 정보를 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show.md:7]:
```suggestion
차트의 정보를 보여준다
```

File: content/ko/docs/helm/helm_show_all.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Helm Show All"
+---
+
+## helm show all
+
+차트의 모든 정보를 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_all.md:7]:
```suggestion
차트의 모든 정보를 표시한다
```

File: content/ko/docs/helm/helm_show_chart.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Helm Show Chart"
+---
+
+## helm show chart
+
+차트의 정의를 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_chart.md:7]:
```suggestion
차트의 정의를 보여준다
```

File: content/ko/docs/helm/helm_show_all.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+### 개요
+
+
+이 명령어는 차트(디렉터리, 파일 또는 URL)를 검사하고 모든 내용을 
+표시한다. (values.yaml, Charts.yaml, README)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_show_all.md:13]:
```suggestion
이 명령어는 차트(디렉터리, 파일 또는 URL)를 검사하고 모든 내용을 
보여준다. (values.yaml, Charts.yaml, README)
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다

**Comment by jmnote** [comment]
lgtm

**Comment by singlemocha** [comment]
LGTM

File: content/ko/docs/helm/helm_show.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_show.md:38]: Why is this default changing from what's in the English docs?

File: content/ko/docs/helm/helm_show_all.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_show_all.md:53]: Same question

File: content/ko/docs/helm/helm_show_chart.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_show_chart.md:52]: Same question

File: content/ko/docs/helm/helm_show_readme.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_show_readme.md:53]: Same question

File: content/ko/docs/helm/helm_show_values.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_show_values.md:52]: Same question

File: content/ko/docs/helm/helm_show.md (hunk 8 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_show.md:38]: I will modify every I PRed as your comment like issue #810 :D

File: content/ko/docs/helm/helm_show_all.md (hunk 9 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_show_all.md:53]: I will modify every I PRed as your comment like issue #810 :D

File: content/ko/docs/helm/helm_show_chart.md (hunk 10 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_show_chart.md:52]: I will modify every I PRed as your comment like issue #810 :D

File: content/ko/docs/helm/helm_show_readme.md (hunk 11 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_show_readme.md:53]: I will modify every I PRed as your comment like issue #810 :D

File: content/ko/docs/helm/helm_show_values.md (hunk 12 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_show_values.md:52]: I will modify every I PRed as your comment like issue #810 :D

**Review by bridgetkromhout — APPROVED** [review]
LGTM!

**Pull request merged**